api/v1alpha1/ndb_api_helpers.go

```diff
@@ -227,6 +292,11 @@ func GetDatabasePortByType(dbType string) int32 {
 	}
 }
 
+func isEmptyProfile(customProfile Profile) (isEmpty bool) {
+	isEmpty = customProfile == (Profile{})
+	return
+}
```

Comment:
Although this function has the correct functionality, I feel we can change the argument to say `profile` instead of `customProfile`. This function will work for any kind of profiles and we might not want add semantic meaning to the parameter name by prefixing it with 'custom'.